packages/fx-core/src/plugins/resource/frontend/env.ts

```diff
@@ -98,7 +98,7 @@ async function _saveEnvFile(envPath: string, envs: RemoteEnvs, configs: RemoteEn
   await fs.ensureFile(envPath);
 
   const envString =
-    `# Following variables are generated by TeamsFx${os.EOL}` +
+    `# TeamsFx will overwrite the following variable values when running debug${os.EOL}` +
```

Comment:
Maybe `TeamsFx will overwrite the following variable values when provisioning`?

Reply:
Revert this file since it's for provision.